Funcky.Xunit/Extensions/ToTheoryDataExtension.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Funcky.Xunit
+{
+    public static class ToTheoryDataExtension
+    {
+        public static TheoryData<T1> ToTheoryData<T1>(this IEnumerable<T1> enumerable)
```

Comment:
Methods can be marked as `[Pure]`

Reply:
public interface declared as pure: ba0eab7d